rfc/text/advanced/rpc_progressive_call_invocations.md

```diff
@@ -20,16 +20,16 @@ and *Dealers* (`role := "dealer"`) via
         HELLO.Details.roles.<role>.features.progressive_call_invocations|bool := true
 
 Progressive call invocations can work only if all three peers support and announce this feature.
-In addition, *Callees* MUST announce support of the _Call Cancelling_ feature via
+In addition, *Callees* MUST announce support of the _Call Canceling_ feature via
 
 {align="left"}
-        HELLO.Details.roles.callee.features.call_cancelling|bool := true
+        HELLO.Details.roles.callee.features.call_canceling|bool := true
```

Comment:
please do not change this *attribute / identifier* - 
*Consistency is good, but backward compatibility too*